front/src/common/api/generatedEditoastApi.ts

```diff
@@ -1514,6 +1564,34 @@ export type PostTimetableByIdTrainScheduleApiArg = {
   id: number;
   body: TrainScheduleBase[];
 };
+export type GetTowedRollingStockApiResponse = /** status 200  */ PaginationStats & {
+  results: TowedRollingStock[];
+};
+export type GetTowedRollingStockApiArg = {
+  page?: number;
+  pageSize?: number | null;
+};
+export type PostTowedRollingStockApiResponse =
+  /** status 200 The created towed rolling stock */ TowedRollingStock;
+export type PostTowedRollingStockApiArg = {
+  towedRollingStockForm: TowedRollingStockForm;
+};
+export type GetTowedRollingStockByTowedRollingStockIdApiResponse =
+  /** status 200 The requested towed rolling stock */ TowedRollingStock;
+export type GetTowedRollingStockByTowedRollingStockIdApiArg = {
+  towedRollingStockId: number;
```

Comment:
For the "standard" rolling stock, we use mostly "name" instead of "id" now. Won't we want to do the same with the towed rolling stock ?

Reply:
This change from `id` to `name` has been done in operational studies for the trainschedule endpoints (using `TrainScheduleBase`). But we will probably keep needing the endpoints with `id` for the rolling stock editor and maybe we will want to use towed rs in the editor as well in the future ?

Anyway you argument is totally legit as this is used only for stdcm where we don't use `trainschedule` :)

Reply:
ok, let's leave it for when we implement the rolling stock editor part., if ever.